### DISCUSSION TITLE
Memory Limit

### DISCUSSION THREAD

**Discussion by Madhucrj3** [pyodide/pyodide#3796] [Q&A]
How to restrict memory used by pyodide in web browser

**Comment by rth** [comment]
The current memory limit in Pyodide is 2GB. See https://stackoverflow.com/questions/75559641/memory-limits-in-pyodide for more details.

I'm not aware of an easy way to restrict this. If you can describe your use case in more detail, explaining why you need it,  it would help to get a more detailed response.

**Reply by Madhucrj3** [reply]
i want to run python code in web browser , and i want pyodide to use only around 500mb of browser memory  , if this limit exceded , i want to interrupt the execution of code and according to our use case may be browser may crash if 2GB memory is not available for user.
could you suggest any way to achive this
